codice/ddf
# DDF-2108 Remove dependencies on and disable the Felix console

#### What does this PR do?
Disables the Felix webconsole as well as removes dependencies on it. Those dependencies included some branding capability which had to be refactored since other modules depend on it (landing page, platform ui config, etc.).

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@tbatie 
@coyotesqrl 
@ryeats 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison

#### How should this be tested?
Full build including itests; deploy and verify the felix web console is disabled. Verify branding on pages (admin ui, search ui, landing page) as appropriate. Also verify landing page and platform ui configurations can be changed and are updated correctly.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
DDF-2108

#### Screenshots (if appropriate)
N/A

#### Checklist:
